tutorial/README.md

```diff
@@ -88,35 +86,28 @@ Therefore for larger images, please use `bioformats2raw + raw2ometiff`.
 > by `raw2ometiff` - `LZW`, `zlib`, and `Uncompressed` as well as `jpeg` compression for 8 bit data. Support
 > for JPEG-2000 for >8 bit data is planned. Please open an issue if you would like this more immediately.
 
+If your OME-TIFF image has many [TIFF IFDs](https://en.wikipedia.org/wiki/TIFF#Multiple_subfiles), which correspond to indvidual time-z-channel sub-images, please generate an `offsets.json` file as well for remote HTTP viewing.
+This file contains the byte offsets to each IFD and allows fast interaction with remote data:
+
+ ```bash
+ $ pip install generate-tiff-offsets
+ $ generate_tiff_offsets --input_file Luca-7color_Scan1.ome.tif 
+ ```
+
+For viewing in Avivator, this file should live adjacent to the OME-TIFF file in its folder and will be automatically recognized and used.
+For use with Viv's loaders/layers, you need to fetch the `offsets.json` and pass it in as an argument to the [loader](http://viv.gehlenborglab.org/#createometiffloader).
+Please see [this sample](http://viv.gehlenborglab.org/#getting-started) for help getting started with using the Viv API.
+
 ### Viewing in Avivator
 
 There are a few different ways to view your data in Avivator.
 
-If you have an OME-TIFF or Bio-Formats "raw" Zarr output saved locally, you may simply drag and drop
-the file (or directory) over the canvas or use the "Choose file" button to view your data.
+If you have an OME-TIFF saved locally, you may simply drag and drop
+the file over the canvas or use the "Choose file" button to view your data.
 Note that this action does **NOT** necessarily load the entire dataset into memory. Viv still works as normal and will retrieve data tiles based on the viewport for an image pyramid and/or a specific channel/z/time selection.
```

Comment:
Not sure if this bit is necessary. I think reducing word count is probably valuable.

```suggestion
```